source/_member_template.html.haml

```diff
@@ -5,9 +5,9 @@
       .row.vertical-align
         .col-xs-6.col-md-6
           -if member.img == nil || member.img.length == 0
-            %img.img-fluid.text-center.rounded{src: "img/unknown.jpg"}
+            %img.img-fluid.text-center.rounded{src: "/img/unknown.jpg"}
           - else
-            %img.img-fluid.text-center.rounded{src: "img/members/" + member.img}
+            %img.img-fluid.text-center.rounded{src: "/img/members/" + member.img}
```

Comment:
A próbowałeś `image_tag member.img, class: "img-fluid text-center rounded"`?

Reply:
Nie próbowałem, a to dużo zmienia?